csrankings-k.csv

```diff
@@ -737,6 +737,7 @@ Koen Langendoen,TU Delft,http://www.st.ewi.tudelft.nl/~koen,goNfvF8AAAAJ
 Koen V. Hindriks,VU Amsterdam,http://www.koenhindriks.nl,BpwoE18AAAAJ
 Koh Hosoda,Osaka University,https://www-arl.sys.es.osaka-u.ac.jp/site,5ledxpAAAAAJ
 Kohei Suenaga,Kyoto University,https://www.fos.kuis.kyoto-u.ac.jp/~ksuenaga,rwSrOdwAAAAJ
+Kohitij Kar,York University,https://kohitij.com/,T7ZMcGkAAAAJ

```

Comment:
Not in CS. Please clarify.
https://www.yorku.ca/science/profiles/faculty/kohitij-kar/